tests/testthat/test-int-latent_individual.R

```diff
@@ -68,6 +68,74 @@ test_that("epidist.epidist_latent_individual Stan code has no syntax errors and
   expect_no_error(mod_gamma$compile())
 })
 
+test_that("epidist.epidist_latent_individual fits and the MCMC converges  in the gamma delay case", { # nolint: line_length_linter.
+  skip_on_cran()
+  set.seed(1)
+  
+  gamma_family <- epidist_family(
+    prep_obs_gamma,
+    family = "gamma",
+    dpars = c("mu", "shape"),
+    links = c("log", "log"),
+    lb = c(0, 0),
+    ub = c(NA, NA)
+  )
+
+  gamma_formula <- epidist_formula(prep_obs_gamma)
+  gamma_formula$pforms$sigma <- NULL
+  gamma_formula <- brms::bf(gamma_formula, as.formula("shape ~ 0"))
+  
+  fit_gamma <- epidist(
+    data = prep_obs_gamma,
+    family = gamma_family,
+    prior = brms::prior("normal(0, 3)", class = "Intercept"),
```

Comment:
Have you tried playing around with raw brms and gamma? It looks like it expects the shape to have no link and to use a gamma prior by default? https://rpubs.com/jwesner/gamma_glm

Reply:
Yeah interesting. Here is the current Stan code:

```
> fit_gamma
// generated with brms 2.21.6
functions {
  // code chunks used from epidist 0.0.0.9000

  // Here the strings
// * gamma
// * vector mu, vector shape
// * mu, shape
// are/have been replaced using regex

real latent_gamma_lpdf(vector y, vector mu, vector shape, vector pwindow,
                           vector swindow, array[] real obs_t) {
  int n = num_elements(y);
  vector[n] d = y - pwindow + swindow;
  vector[n] obs_time = to_vector(obs_t) - pwindow;
  return gamma_lpdf(d | mu, shape) - gamma_lcdf(obs_time | mu, shape);
}
}
data {
  int<lower=1> N;  // total number of observations
  vector[N] Y;  // response variable
  // data for custom real vectors
  array[N] real vreal1;
  // data for custom real vectors
  array[N] real vreal2;
  // data for custom real vectors
  array[N] real vreal3;
  int prior_only;  // should the likelihood be ignored?
  int wN;
  array[N - wN] int noverlap;
  array[wN] int woverlap;
}
transformed data {
}
parameters {
  real Intercept;  // temporary intercept for centered predictors
  real Intercept_shape;  // temporary intercept for centered predictors
  vector<lower = 0, upper = 1>[N] swindow_raw;
vector<lower = 0, upper = 1>[N] pwindow_raw;
}
transformed parameters {
  real lprior = 0;  // prior contributions to the log posterior
  vector<lower = 0>[N] pwindow;
vector<lower = 0>[N] swindow;
swindow = to_vector(vreal3) .* swindow_raw;
pwindow[noverlap] = to_vector(vreal2[noverlap]) .* pwindow_raw[noverlap];
if (wN) {
  pwindow[woverlap] = swindow[woverlap] .* pwindow_raw[woverlap];
}
  lprior += normal_lpdf(Intercept | 0, 3);
  lprior += student_t_lpdf(Intercept_shape | 3, 0, 2.5);
}
model {
  swindow_raw ~ uniform(0, 1);
pwindow_raw ~ uniform(0, 1);
  // likelihood including constants
  if (!prior_only) {
    // initialize linear predictor term
    vector[N] mu = rep_vector(0.0, N);
    // initialize linear predictor term
    vector[N] shape = rep_vector(0.0, N);
    mu += Intercept;
    shape += Intercept_shape;
    mu = exp(mu);
    shape = exp(shape);
    target += latent_gamma_lpdf(Y | mu, shape, pwindow, swindow, vreal1);
  }
  // priors including constants
  target += lprior;
}
generated quantities {
  // actual population-level intercept
  real b_Intercept = Intercept;
  // actual population-level intercept
  real b_shape_Intercept = Intercept_shape;
}
```